spack-configs/rocm/packages.yaml

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: (C) The kokkos-fft development team, see COPYRIGHT.md file
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
+
+packages:
+  llvm-amdgpu:
```

Comment:
Given the error, I think there are other packages related to HIP that Spack wants to install.